xDrip/Model/Glucose/GlucoseReading.swift

```diff
@@ -517,4 +525,56 @@ final class GlucoseReading: Object {
             displayDeltaName = displayGlucose.deltaName
         }
     }
+    
+    static func isOptimalConditionToCalibrate() -> Bool {
+        var optimalCalibrationCondition = false
+        let masterReadings = GlucoseReading.allMasterForCurrentSensor
+        if masterReadings.count >= 3 {
```

Comment:
Replace `if`s with `guard`s to improve code readability. And not just on this line, but on the whole function.